test/parallel/sql/utils/common.sql

```diff
@@ -59,3 +63,28 @@ BEGIN
     RETURN found;
 END;
 $$ LANGUAGE plpgsql;
+
+-- Determine if the two  queries provided return the same results
+-- At the moment this only works on queries that return rows with the same entries as one another
+-- if you try to compare uneven numbers of columns or columns of different types it will generate an error
+CREATE OR REPLACE FUNCTION results_match(left_query text, right_query text) RETURNS boolean AS $$
+DECLARE
+    left_cursor REFCURSOR;
+    left_row RECORD;
+
+    right_cursor REFCURSOR;
```

Comment:
there is quite some repetition between this file and the common.sql in non-parallel tests. Can we somehow avoid copying and have the file in one of the two locations?

Reply:
Yeah that makes sense. No reason we can't use one file